i18n/zh/docusaurus-plugin-content-docs/current/user-guide/deployments/configuration.md

```diff
@@ -319,7 +346,7 @@ experimental_write_cache_path = "/var/data/s3_write_cache"
 experimental_write_cache_size = "5GiB"
 ```
 
-`experimental_write_cache_path` 的默认值是 `{data_home}/object_cache/write`。
+`experimental_write_cache_path` 默认值位于 `{data_home}` 目录下。
```

Comment:
`{data_home}/cache` would be more precise.

Reply:
Since `v0.11.2` the default value for `experimental_write_cache_path` is `{data_home}`. The engine appends `cache` to the data home.